twilight-model/src/gateway/reaction.rs

```diff
@@ -127,14 +127,6 @@ impl<'de> Visitor<'de> for ReactionVisitor {
 
         tracing::trace!(?channel_id, ?emoji, ?message_id, ?user_id);
 
-        let member = if let (Some(guild_id), Some(member)) = (guild_id, member) {
-            tracing::trace!(%guild_id, ?member, "setting member guild id");
-
-            Some(member.into_member(guild_id))
-        } else {
-            None
-        };
-
```

Comment:
This whole manual implementation can now be replaced with `#[derive(Deserialize)]` (part of #1364)